kolibri/core/assets/src/views/sync/SelectAddressModalGroup/api.js

```diff
@@ -48,26 +48,28 @@ function fetchAddressesForLOD(LocationResource = NetworkLocationResource) {
 }
 
 function fetchAddressesWithChannel(withChannelId = '', LocationResource = NetworkLocationResource) {
-  return LocationResource.fetchCollection({ force: true }).then(locations => {
```

Comment:
Do we need to do this filtering for channel import? It's only facility syncing that is problematic here.

Reply:
I thought we would prefer  not to use them for any sync work, but if you think it's fine to import from them, no problem. I've removed it

Reply:
Yeah, I just think of situations where there are only other SoUD devices on a network but you still want to get content peer to peer.